thinkcompany/think-ui-library
# SWIM-603 Update Media Only component (fka Image Only)

### Feature Status

### Description of Changes
- Add video source and controls to toggle media types
- Rename component to "Media Only"

[SWIM-603](https://thinkcompany.atlassian.net/browse/SWIM-603)

#### Screenshots
![Screen Shot 2021-08-30 at 11 37 21 AM](https://user-images.githubusercontent.com/1825366/131365384-a3514ffe-9519-4025-b9db-88745fceae7e.png)


### How to Review
1. Check out the Netifly [deploy preview](https://deploy-preview-135--think-ui-library.netlify.app/?path=/story/storybook-welcome--page)
2. Click on "Media Only" listed under "Components"
3. Toggle the control called "media type" to ensure the appropriate media type is being displayed. 

Note: the 'vertical offset' control is only supposed to work with an image. 


### Merge Checklist:
